### ISSUE TITLE
woql_as(), tuple index out of range

### ISSUE THREAD

**Issue by k-tipp** [terminusdb/terminusdb-client-python#159]
**Describe the bug**
An IndexError is thrown, if woql_as() is used as shown in the documentation (see [here](https://terminusdb.com/docs/how-tos/python/import-data-python/#turtle--rdf)).

**To Reproduce**
Execute:
```
[x,y,z] = WOQLQuery().vars("x","y","z")
query = WOQLQuery().get(
        WOQLQuery().woql_as(x).woql_as(y).woql_as(z)
).post("myfile.ttl",
       {"type" : "turtle"})
```

**Expected behavior**
No exception

**Error logs**
![grafik](https://user-images.githubusercontent.com/3494491/98871614-7ead6b00-2475-11eb-9b51-0551ee1c9ff3.png)

**System information (please complete the following information):**
 - OS: Ubuntu 20.04.1 LTS (Windows Subsystem for Linux)
 -  terminus-client-python version: tested with 0.3.1 and 0.4.0

**Comment by k-tipp** [comment]
Found a workaround [here](https://github.com/terminusdb/terminusdb-tutorials/blob/master/wordnet/loading_wordnet.py).
```
ttl_file = open("./myfile.ttl")
contents = ttl_file.read()
ttl_file.close()
client.insert_triples(
        "schema","main",
    contents,
    "Adding WordNet in chunks ()")
```

**Comment by luke-feeney** [comment]
Thanks for this @k-tipp  - we are going to discuss this at the next issues and docs triage session in the Discord server (thou the workaround looks good).

What: Issues and Docs Triage
Where: Discord Server (https://discord.gg/CzE5yZdjpr)
When: Every Thursday from this week until the end of time @ 2pm GMT; 3pm CET; 9am ET; 6am PT; 7.30pm IST

Will report back after or come along to represent the bug! (The squeakiest wheel will get all the oil)

**Comment by luke-feeney** [comment]
This is a bug in the python client - we are working on it and will report back.

**Comment by Cheukting** [comment]
It is now fixed in v0.5.1
